src/Entity/Video.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-declare(strict_types=1);
+declare (strict_types = 1);
```

Comment:
Please revert :)

Reply:
commit 455c3cb26c2e1f951e9659c44c90b70f065d1a7c

Reply:
ehh. My IDE changed it again. commit 10bf12d47c63ca6116839fcc06cce1516c1925b7 should be ok.